samples/matter/light_bulb/child_image/mcuboot_multi_image_matter_dfu.conf

```diff
@@ -23,5 +22,24 @@ CONFIG_FLASH_SIMULATOR_DOUBLE_WRITES=y
 CONFIG_FLASH_SIMULATOR_STATS=n
 
 # Enable custom command to erase settings partition.
-CONFIG_ENABLE_MGMT_PERUSER=y
-CONFIG_BOOT_MGMT_CUSTOM_STORAGE_ERASE=y
+# These configs are valid only without bootloader size optimization!
+# To use it enable CONFIG_SERIAL=y, CONFIG_UART_CONSOLE=y and CONFIG_CONSOLE=y
+# CONFIG_ENABLE_MGMT_PERUSER=y
+# CONFIG_BOOT_MGMT_CUSTOM_STORAGE_ERASE=y
+
+
+# bootloader size optimization
+CONFIG_LOG=n                        # ~ 5204 B
+CONFIG_BOOT_BANNER=n                # ~ 80 B
+CONFIG_CONSOLE=n                    # ~ 1212 B
+CONFIG_GPIO=n                       # ~ 1668 B
+CONFIG_NO_RUNTIME_CHECKS=y          # ~ 64 B
+CONFIG_PRINTK=n                     # ~ 1296 B
```

Comment:
I believe that one is not needed, as most likely it is enabled by `select PRINTK` in `LOG`, so if you set `LOG=n` the `PRINTK` will be disabled either. You could check if all of those settings here need to be disabled or maybe some of them are already disabled by the other options, but it's just a hint.

Reply:
I removed that line and nothing changed with the bootloader size, so I think it is disabled.